### DISCUSSION TITLE
Can I finetune model to korean, (especially medical) part?

### DISCUSSION THREAD

**Discussion by KimChanYoup** [deepgram/community#824] [General help] [answered]
(As I am using a translator, there might be some imperfect English in my message)

Hello. I am a student studying hard in the AI field in Korea.

When I received a project to support the medical sector through STT, I thought it would be great to make recording conversations the main theme of the project.

While researching information, I learned that the best quality STT service is provided by Deepgram. I realized that this service is overwhelmingly the best due to its incredible quality and unbeatable pricing policy, so I decided to use Deepgram without hesitation.

This project even has the potential to be commercialized, so I am studying Deepgram even more diligently!

What I am curious about is, although I have confirmed through the tutorial that it recognizes Korean well enough, this is based on standard Korean. I am unsure whether the STT service will work well with regional dialects or difficult medical terms.

So, I thought about obtaining regional dialect datasets or medical terminology datasets for additional training (fine-tuning).

I wonder if there is any documentation on this part (since I am Korean, I haven't been able to completely analyze the docs yet 😭).

Or, if there are other methods, I would like to know about them.

I believe you will remain the top service and the overwhelming leader in the STT field, and I hope your company continues to develop day by day!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
There isn't a self-serve way of doing this, and I am unsure if that is in the plans to explore that. However, if you have a need, we do have a "white glove" kind of service to fine-tune models to your data.

**Reply by madhan-palani-zoomrx** [reply]
Hi @dvonthenen 

I hope you are doing well! I would like to understand the details of the service "fine-tune models to your data" but I couldn't find any related resources. Can you share the details of the same? 

Thanks in advance.

**Reply by davidvonthenen** [reply]
hi @madhan-palani-zoomrx 

"white glove" means you need to speak to a sales person. they will talk about the process, talk about collecting data, the team will build the model, etc. If you are interested, I can put you in contact with a sales rep.

**Reply by madhan-palani-zoomrx** [reply]
Thanks, @dvonthenen. 
I have reached out to Juan Salazar Gasca over the mail and discussed some specifics. I will take it from here and let you know if anything else is required. I appreciate your effort.

**Reply by davidvonthenen** [reply]
All good!
